frontend/package.json

```diff
@@ -23,11 +23,14 @@
     "react-redux": "^7.2.3",
     "react-router-dom": "^5.2.0",
     "react-scripts": "4.0.3",
-    "typescript": "^4.2.4"
+    "superagent": "^6.1.0",
+    "typescript": "^4.2.4",
+    "uuid": "^8.3.2"
   },
   "resolutions": {
     "react": "^17.0.2",
-    "react-dom": "^17.0.2"
+    "react-dom": "^17.0.2",
+    "three": "0.111.0"
```

Comment:
Needed because Geppetto Canvas doesn't work with the latest. 
I don't remember why it is 0.111.0 specifically but it was like that in geppetto-meta package.json

Reply:
okay seems fine